scripts/processsourcefiles.sh

```diff
@@ -27,7 +27,10 @@ then
   git clone --quiet -b main https://github.com/"$FORK".git content/en
   echo 'Getting pre-release development docs from main branch'
   # Move "pre-release" docs content into the 'development' folder:
-  mv content/en/docs content/en/development
+  
+  # /development now is going to be redirected to https://beta-knative.netlify.app/development
+  # mv content/en/docs content/en/development
+
```

Comment:
Sorry, I didnt mean to complicate this.

Is the script still happy if HEAD from `main` is in the /docs folder (thats where we add the `release-vLATEST` stuff)? I remember having trouble moving things around when files already existed: https://github.com/csantanapr/knative-website/blob/redirect-dev/scripts/processsourcefiles.sh#L49

for example:
```suggestion
  # instead, make sure the /docs folder is empty so that the latest release can be add there
  rm -rf content/en/docs
```

Reply:
@RichieEscarez to be frank the scrip is too complicated for me to seat down and figure out what is suppose to do.
Also I tried to run a local build and again I can't run it locally hugo complains about some HTML file

Reply:
So I'm looking for the easiest way to not have the /development folder present

Reply:
I'm happy if you want to take over today and make the changes

Reply:
i tested it locally. we do need to apply my suggestion